client/login/magic-login/style.scss

```diff
@@ -893,7 +893,7 @@
 	margin-bottom: 32px;
 
 	.form-text-input.magic-login__verify-code-character-field {
-		width: 61px;
+		max-width: 61px;
```

Comment:
Is it worth using a relative unit here like `rem` instead of `px`?

Reply:
Thanks for checking. I don't think the unit is important here. I was also considering removing `width` completely, because flex handles it and flex gap together with max container width makes sure the size isn't greater than what we want. I kept it just in case to avoid last minute changes.
I'm not keen to change it to rems to avoid recalculating the values when most likely it won't be used.

Reply:
Great. Thanks